tools/github/contributions.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+parse_args() {
+    if [ -z "$2" ]; then
+        echo "value for parameter '$1' cannot be empty"
+        exit 1
+    fi
+    case "$1" in
+        -t)
+            prtype="$2"
+            ;;
+        -n)
+            newbranch="$2"
+            ;;
+        -r)
+            remote="$2"
+            ;;
+        -b)
+            contributorbranch="$2"
+            ;;
+        -c)
+            connector="$2"
+            ;;
+        *)
```

Comment:
And then add a section for `--help|-h)` that prints each of these options and their descriptions so the user doesnt have to read this script every time they run it to remember the flags